driver/normalizer/annotation.go

```diff
@@ -356,20 +372,17 @@ var Annotations = []Mapping{
 		"lines": {Arr: true, Roles: role.Roles{role.Noop}},
 	}, role.Noop),
 
-	AnnotateType("NoopLine", FieldRoles{
-		"noop_line": {Rename: uast.KeyToken},
-	}, role.Noop, role.Comment),
-
-	AnnotateType("NoopSameLine", FieldRoles{
-		"s": {Rename: uast.KeyToken},
-	}, role.Noop, role.Comment),
+	AnnotateType("NoopLine", MapObj(Obj{
+		"noop_line": UncommentBashLike("txt"),
```

Comment:
But Annotated should not change values. All other drivers leave it with `#`, and this was asked by ML - preserve all node tokens in the `@token`, so it can match the positional info.